frontend/src/types/models/robot.ts

```diff
@@ -0,0 +1,13 @@
+import { BaseModel } from ".";
+
+export interface Robot extends BaseModel {
+  name: string;
+  position?: string;
```

Comment:
This is fine for now, just keep in mind that this might be changed later to coordinates or something similar.